suave/builder/session_manager.go

```diff
@@ -37,6 +39,38 @@ type Config struct {
 	MaxConcurrentSessions int
 }
 
+func getEnvAsUint64(key string, defaultVal uint64) uint64 {
+	valStr := os.Getenv(key)
+	if val, err := strconv.ParseUint(valStr, 10, 64); err == nil {
+		return val
+	}
+	return defaultVal
+}
+
+func getEnvAsInt(key string, defaultVal int) int {
+	valStr := os.Getenv(key)
+	if val, err := strconv.Atoi(valStr); err == nil {
+		return val
+	}
+	return defaultVal
+}
+
+func getEnvAsDuration(key string, defaultVal time.Duration) time.Duration {
+	valStr := os.Getenv(key)
+	if val, err := time.ParseDuration(valStr); err == nil {
```

Comment:
it's not great if errors are silently ignored. it should probably panic or return an error if there's an error in parsing the user-supplied string

Reply:
yeap, thank you! fixed